stackoverflow on fabric 1.18.2 server

log:
```
[15:59:20] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.class_148: Exception ticking world
        at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:911) ~[server-intermediary.jar:?]
        at net.minecraft.class_3176.method_3813(class_3176.java:322) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:851) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.handler$bjd000$modifiedRunLoop(MinecraftServer.java:5896) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:676) ~[server-intermediary.jar:?]
        at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:273) ~[server-intermediary.jar:?]
        at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.StackOverflowError
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStandSpeedEnchant.method_20742(BrewingStandSpeedEnchant.java:38) ~[%5B%EF%BF%BD%EF%BF%BD%D0%AF%EF%BF%BD%EF%BF%BD%EF%BF%BD%EF%BF%BD%5D%20PortableCraft-fabric-1.18.2-1.2.0.jar:?]
        at committee.nova.portablecraft.common.enchants.BrewingStand[15:59:20] [Server thread/INFO]: Saving worlds
```
modlist:
![image](https://user-images.githubusercontent.com/18550358/169685112-9670ec11-6633-48d7-9e28-0baed872fbf2.png)
![image](https://user-images.githubusercontent.com/18550358/169685134-1844185d-f2e3-4158-b244-f2ada5d6f43b.png)

--- a/.gitignore
+++ b/.gitignore
@@ -1,0 +1,2 @@
+/.gradle/
+/.idea/

--- a/src/main/java/committee/nova/portablecraft/common/enchants/BrewingStandSpeedEnchant.java
+++ b/src/main/java/committee/nova/portablecraft/common/enchants/BrewingStandSpeedEnchant.java
@@ -35,7 +35,7 @@ public class BrewingStandSpeedEnchant extends BaseEnchant {
 
     @Override
     public int getMaxPower(int pEnchantmentLevel) {
-        return this.getMaxPower(pEnchantmentLevel) + 20;
+        return this.getMinPower(pEnchantmentLevel) + 20;
     }
 
     @Override

